src/garage/tf/policies/task_embedding_policy.py

```diff
@@ -0,0 +1,332 @@
+"""Policy class for Task Embedding envs."""
+import abc
+
+import akro
+import numpy as np
+
+from garage.tf.policies import StochasticPolicy
+
+
+class TaskEmbeddingPolicy(StochasticPolicy):
```

Comment:
consider whether you can make this a general ContextConditionedPolicy as in torch, where the caller provides a context encoder (GaussianMLPEncoder) and a policy class (GaussianMLPPolicy).

Reply:
that is definitely the right way to eliminate repeating code, if we have a second context conditioned algorithm. but let's refrain from making this abstraction until we really need it

Reply:
we already have many context-conditioned algorithms (MTSAC, MTPPO, MTTRPO, etc.) but I am willing to settle for filing an issue here.

Reply:
Oped an issue at https://github.com/rlworkgroup/garage/issues/1334